terraform/modules/ecs_module/security-groups.tf

```diff
@@ -0,0 +1,75 @@
+resource "aws_security_group" "aws-lb" {
+  name        = "${var.nginx_test_name}-load-balancer"
+  description = "${var.nginx_test_name}"
+  vpc_id      = aws_vpc.aws-vpc.id
+  ingress {
+    protocol    = "tcp"
+    from_port   = var.nginx_port
+    to_port     = var.nginx_port
+    cidr_blocks = var.app_sources_cidr
+  }
+  egress {
```

Comment:
📝 **[tfsec]** <[AVD-AWS-0124](https://aquasecurity.github.io/tfsec/v1.27.6/checks/aws/ec2/add-description-to-security-group-rule/)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Security group rule does not have a description.